### PR TITLE
Typo fix

### PR DESCRIPTION
**Pull request by NekoJonez**
Fizzy should become fuzzy. 
Also, tested on WP 6.0.2 and working flawlessly.

### CODE REVIEW

**Comment by pedro-mendonca** [comment]
There is no need to bump the X.X.x, the compatibility check is by X.X version, and the compatibility notice automatically adds the x.x.X point release.
https://wordpress.org/plugins/force-update-translations/
imagem

File: readme.txt [hunk 1]
```diff
@@ -2,7 +2,7 @@
 Contributors: mayukojpn, nao, dartui, pedromendonca, casiepa, mekemoke, miyauchi, nekojonez
 Tags: translation
 Requires at least: 4.9
-Tested up to: 6.0
+Tested up to: 6.0.2
```
Comment (pedro-mendonca) [inline readme.txt:5]: No need to bump the point release, should be left as 6.0, as there are no breaking changes in point releases.

**Comment by NekoJonez** [comment]
Since, I don't really know how to undo something for a PR... I'll close this one and make a new one to fix the typo. :) Thanks @pedro-mendonca for that tip :D

**Comment by pedro-mendonca** [comment]
You can add another commit reverting the 6.0.2 to 6.0 and it changes the PR, give it a try :)

**Comment by NekoJonez** [comment]
Too late now :P, I deleted the branch & created the new PR   But I'll keep that in mind for the next time. 

Today I learned.

**Pull request closed without merging**